### ISSUE TITLE
Exception: No perl executable found. Camera Raw Previews app will not work.

### ISSUE THREAD

**Issue by thenameiwantedwastaken** [ariselseng/camerarawpreviews#61]
I get the above error in log files on nextcloud.

This is a docker install of nextcloud based on linuxserver/nextcloud. I am on an Odroid XU4 (ARM processor). Nextcloud is 20.0.3.

Any tips on how to solve this?

**Comment by ariselseng** [comment]
You will need perl on arm. I recommend you use the official arm image.

**Comment by thenameiwantedwastaken** [comment]
I'm sorry, but I don't understand your answer. I do understand that I need perl, but what exactly is the official arm image that you refer to? Is it a docker image? Is it an image of nextcloud, or of perl?

**Comment by ariselseng** [comment]
I am sorry. I meant the official nextcloud docker image: https://hub.docker.com/_/nextcloud

That one has perl, atleast on x86.

**Comment by thenameiwantedwastaken** [comment]
Okay - thanks for the quick response. I don't really want to change to a completely different nextcloud image for fear or breaking my existing setup. :-( Thanks anyway. I wonder why the linuxserver nextcloud image does not include perl.
